backend/src/main/java/de/neuefische/backend/service/UserService.java

```diff
@@ -17,17 +18,21 @@ public UserService(UserRepo userRepo, UuidGeneratorService uuidGeneratorService)
     }
 
     public List<User> listUsers() {
-        return userRepo.getUsers();
+        return userRepo.findAll();
     }
 
     public User addUser(User user){
         User userWithId = new User(this.uuidGeneratorService.generateUuid(),user.name(),user.visitedCountries());
-        return userRepo.addUser(userWithId);
+        return userRepo.save(userWithId);
     }
 
 
     public String deleteUser(String id){
-        userRepo.deleteUser(id);
+        Optional<User> byId = userRepo.findById(id);
```

Comment:
Hint: There's a really nice wrapper for cases such as this:

```java
userRepo.findById(id).orElseThrow(() -> new NoSuchElementException("There is no user with the requested ID " + id))
```

Reply:
My actual issue has been the naming of `byId` -> feel free to use the syntax above or improve the naming of this variable (what is `byId`  supposed to mean?)